pkg/api/soc.go

```diff
@@ -155,6 +157,15 @@ func (s *Service) socUploadHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	encryptedReference := sch.Address()
```

Comment:
Maybe have `schAddress` instead of `encryptedReference` ?

Reply:
renamed to reference